docs/src/main/paradox/features/theming.md

```diff
@@ -0,0 +1,93 @@
+Theming
+=======
+
+Paradox uses "theming" to generate its target files. Combined with templates within it, themes can produce a global styling of the site generated.
```

Comment:
A theme is a collection of templates and other resources like css and image files that can be distributed as a web jar. Using this mechanism, multiple Paradox sites can have consistent look and feel.